### PR TITLE
Async Task for Resize

### PR DESCRIPTION
**Pull request by cristianoccazinsp**
Move resize operation to a parallel async task in order to avoid blocking React's bridge.

### CODE REVIEW

**Comment by cristianoccazinsp** [comment]
Forgot to mention, this is Android only change.

**Comment by gmaclennan** [comment]
We have been using this fork in our beta app https://github.com/digidem/mapeo-mobile and it has been working well without problems. I support merging this @Minishlink

**Comment by cristianoccazinsp** [comment]
Any updates? Could really have this merged

**Comment by PierreCapo** [comment]
@cristianoccazinsp Sorry for the delay. Gonna try to review this asap

File: android/src/main/java/fr/bamlab/rnimageresizer/ImageResizer.java [hunk 1]
```diff
@@ -232,7 +232,7 @@ private static Bitmap loadBitmapFromFile(Context context, Uri imageUri, int newW
         // Set a sample size according to the image size to lower memory usage.
         options.inSampleSize = calculateInSampleSize(options, newWidth, newHeight);
         options.inJustDecodeBounds = false;
-        System.out.println(options.inSampleSize);
+        //System.out.println(options.inSampleSize);
```
Comment (PierreCapo) [inline android/src/main/java/fr/bamlab/rnimageresizer/ImageResizer.java]: 👌

**Comment by cristianoccazinsp** [comment]
I've added a new commit that adds width and height (in pixels) to the returned data. 

iOS was quite straight forward, while Android required a few organization updates (return bitmap instead of file, and handle file later)

**Comment by PierreCapo** [comment]
Thanks a lot for the pr and all the work done @cristianoccazinsp . This is available on `v1.1.0`.

**Pull request merged**